nestor_api/lib/config.py

```diff
@@ -68,6 +68,18 @@ def get_project_config(config_path: str = Configuration.get_config_path()) -> di
     return _resolve_variables_deep(project_config)
 
 
+def get_deployments_configs(project_config: dict) -> list:
```

Comment:
```suggestion
def get_deployments(project_config: dict) -> list:
```

Otherwise you will end-up with `config. get_deployments_configs()` calls which are a bit strange IMO